app/storage/metadata_parser.py

```diff
@@ -121,22 +119,6 @@ def validate_schema_name(self, data, **kwargs):
                     "Either 'schema_name' or 'survey' and 'form_type' and 'region_code' must be defined"
                 )
 
-    @post_load
```

Comment:
The `validate_schema_name` function above this needs to be adapted to validate for `eq_id` and `form_type`.

Reply:
Refactored now, so it validates eq_id and form_type, thanks!